bin/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Lets stick to a simplere "#!/bin/bash"

Reply:
IIRC It does not work on Mac.
This is the best way to have a portable script

Reply:
/usr/bin/env is no more guarantee than /bin/bash (since you use Mac it is stricly better only for your use case). Also note that, using env, complicates passing arguments and/or calling the script from cron.
Since this is a dev script i'm ok using env anyway